diff --git a/Cargo.toml b/Cargo.toml
index 7f85b10..2abf2e0 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -3,7 +3,7 @@ members = ["crates/*"]
 
 [workspace.package]
 version = "0.11.10"
-rust-version = "1.92"
+rust-version = "1.91"
 edition = "2024"
 description = "A CalDAV server"
 documentation = "https://lennart-k.github.io/rustical/"
diff --git a/Dockerfile b/Dockerfile
index 0785dc2..21e7830 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM rust:1.92-alpine AS chef
+FROM --platform=$BUILDPLATFORM rust:1.91-alpine AS chef
 
 ARG TARGETPLATFORM
 ARG BUILDPLATFORM
